doc/install_ubuntu.txt

```diff
@@ -12,6 +12,31 @@ For NVIDIA Jetson TX1 embedded platform:
     sudo apt-get install python-numpy python-scipy python-dev python-pip python-nose g++ libblas-dev git
     pip install --upgrade --no-deps git+git://github.com/Theano/Theano.git --user  # Need Theano 0.8(not yet released) or more recent
 
+For Ubuntu 16.04 with cuda 7.5
+
+.. code-block:: bash
+
+    sudo apt-get install python-numpy python-scipy python-dev python-pip python-nose g++ libopenblas-dev git
+    sudo pip install Theano
```

Comment:
For what it worths installing Theano in virtualenv is almost as easy: there is no need for python-scipy and python-numpy apt packages because scipy and numpy are installed from manylinux wheels by default if a recent pip version is used; I think they link against openblas. 16.04 has python-pip recent enough.